docs/add-ins/addin-tutorial.md

```diff
@@ -141,80 +253,63 @@ Locate the line in the manifest that reads `</DesktopFormFactor>`. Above this li
-   ```
-1. Change the `DefaultValue` attribute of the `bt:String` element with `id="groupLabel"` to `Git the Gist`.
+    ```
+1. Within the collection of child elements under the `bt:ShortStrings` element, locate the child element with `id="groupLabel"`.  Change the `DefaultValue` attribute of the this element to **Git the gist**.
+
+    ```xml
+    <bt:String id="groupLabel" DefaultValue="Git the gist"/>
+    ```
 
-   ```xml
-    <bt:String id="groupLabel" DefaultValue="Git the Gist"/>
-   ```
 1. Add the following elements as children of the `bt:ShortStrings` element.
 
-   ```xml
-    <bt:String id="insertGistLabel" DefaultValue="Insert Gist">
-      <bt:Override Locale="es-ES" Value="Inserte el Gist"/>
-    </bt:String>
-    <bt:String id="insertGistTitle" DefaultValue="Insert Gist">
-      <bt:Override Locale="es-ES" Value="Inserte el Gist"/>
-    </bt:String>
-    <bt:String id="insertDefaultGistLabel" DefaultValue="Insert Default Gist">
-      <bt:Override Locale="es-ES" Value="Inserte el Gist predeterminado"/>
-    </bt:String>
-    <bt:String id="insertDefaultGistTitle" DefaultValue="Insert Default Gist">
-      <bt:Override Locale="es-ES" Value="Inserte el Gist predeterminado"/>
-    </bt:String>
-   ```
+    ```xml
+    <bt:String id="insertGistLabel" DefaultValue="Insert gist"/>
+    <bt:String id="insertGistTitle" DefaultValue="Insert gist"/>
+    <bt:String id="insertDefaultGistLabel" DefaultValue="Insert default gist"/>
+    <bt:String id="insertDefaultGistTitle" DefaultValue="Insert default gist"/>
+    ```
+
 1. Add the following elements as children of the `bt:LongStrings` element.
 
-   ```xml
-    <bt:String id="insertGistDesc" DefaultValue="Displays a list of your Gists and allows you to insert their contents into the current message">
-      <bt:Override Locale="es-ES" Value="Muestra una lista de sus Gists y permite insertar su contenido en el mensaje actual"/>
-    </bt:String>
-    <bt:String id="insertDefaultGistDesc" DefaultValue="Inserts the contents of the Gist you mark as default into the current message">
-      <bt:Override Locale="es-ES" Value="Inserta el contenido de lo Gist que marca como predeterminado en el mensaje actual"/>
-    </bt:String>
-   ```
+    ```xml
+    <bt:String id="insertGistDesc" DefaultValue="Displays a list of your gists and allows you to insert their contents into the current message"/>
+    <bt:String id="insertDefaultGistDesc" DefaultValue="Inserts the contents of the gist you mark as default into the current message"/>
+    ```
 
-This defines the string values that will be used for the add-in. It also localizes the strings into Spanish by providing a `bt:Override` element for each string. Additional languages can be added as additional `bt:Override` elements. The `DefaultValue` is used for clients that use the locale specified in the `DefaultLocale` element in the manifest.
+1. Save your changes to the manifest. 
```

Comment:
The "Specify a support page" section contains no instruction to save your changes to the manifest, because the very next section after that ("Specify basic information") describes additional manifest changes, and the final step in that section concludes with "..., and save the file."

![image](https://user-images.githubusercontent.com/3019529/49548360-4d47c000-f89a-11e8-9dc9-4de8312b8f61.png)